worker/Dockerfile.template

```diff
@@ -55,10 +33,6 @@ RUN sed -i 's/^#cgroup_controllers =.*/cgroup_controllers = []/g' /etc/libvirt/q
 
 COPY --from=go-build /go/bin/glider /usr/local/bin
 
-COPY --from=firmware /src/usbsdmux/dist/usbsdmux-configure /usr/local/bin
-COPY --from=firmware /src/teensy_loader_cli/teensy_loader_cli /usr/local/bin
-COPY firmware/StandardFirmataPlus.ino.hex .
-COPY firmware/SDcardSwitch.ino.hex .
```

Comment:
What were these binaries used for previously?

Reply:
I think they were used for the very first iteration of the testbot/autohat board many years ago.